package.json

```diff
@@ -36,10 +41,10 @@
     "eslint": "^8.24.0"
   },
   "scripts": {
-    "lint": "eslint --ext .js,.mjs src pcsync.js pcwatch.js"
+    "lint": "./node_modules/.bin/eslint --ext .js,.mjs src bin"
```

Comment:
You shouldn't need to specify the path to `eslint`. Why is this necessary?

Reply:
Reverted.